data-raw/ictr_data.R

```diff
@@ -3,10 +3,8 @@ library(readr)
 library(usethis)
 
 # The raw data comes from tiltIndicator#87, which includes pre-processing
-ictr_inputs <- read_csv(here("data-raw", "ictr", "ictr_inputs.csv"), show_col_types = FALSE) |>
-  select(all_of(ictr_inputs_crucial()))
+ictr_inputs <- read_csv(here("data-raw", "ictr", "ictr_inputs.csv"))
 use_data(ictr_inputs, overwrite = TRUE)
 
-ictr_companies <- read_csv(here("data-raw", "ictr", "ictr_companies.csv"), show_col_types = FALSE) |>
```

Comment:
The guidelines worked for me. I read then re-wrote the few columns we care about. This way the raw data is lighter and also the code to read it is much simpler.

I intentionally want to see the messages from `read_csv()` to make sure it's using the correct delimiter, decimal point, and column types.

Reply:
@maurolepore Renaming the columns using `rename` function worked at the end and I could finally be able to read the updated csv file with `read_csv` function. Btw the `read_csv2` also did not work if I update the csv file manually. 

Interesting fact: I can now read the csv files (you uploaded in your recent commits) with `read_csv` function even if I make the changes in the file manually. This is some weird behaviour of csv files I don't understand! Nevertheless, it's good at the end that the problem is solved. :)